src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  background-color: black;
+  margin: 0%;
+  padding: 20px;
+}
+
+.outer-box {
+  box-sizing: border-box;
+  width: 350px;
+  background-color: #308dc0;
+  border: 1px solid #e75551;
+  padding: 10px;
+  margin-bottom: 40px;
+}
+
+.middle-box {
+  box-sizing: border-box;
+  background-color: #ffa500;
+  border: 1px solid #e75551;
+  padding: 10px;
+}
+
+.inner-box {
+  box-sizing: border-box;
+  background-color: #ddd;
+  border: 1px solid #e75551;
+  padding: 10px;
+}
```

Comment:
box-sizing, border and padding properties are the same in all three classes, move it to separate class `block`